packages/core/src/Services/socket-general.js

```diff
@@ -141,7 +142,9 @@ const BinarySocketGeneral = (() => {
             case 'Fiat2CryptoTransferOverLimit':
                 // if there is fiat2crypto transfer limit error, we need to refresh the account_status for authentication
                 if (msg_type === 'transfer_between_accounts') {
-                    ui_store.toggleAccountTransferLimitModal(true);
+                    ui_store.toggleAccountTransferLimitModal(true, {
```

Comment:
We can remove the code on `L101` and just use the below instead to follow the same convention as other API error message handlers:
```suggestion
                    ui_store.toggleAccountTransferLimitModal({ state_change: true, 
```